harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -980,6 +980,36 @@ def evaluate_full_dataset(self, data_loader: torch.utils.data.DataLoader) -> Dic
         """
         pass
 
+    def calculate_batch_length(self, batch: Any) -> int:
```

Comment:
rename it to use "get" since it doesn't really calculate anything?

```suggestion
    def get_batch_length(self, batch: Any) -> int:
```